dataland-frontend/src/components/resources/landingPage/AuthSection.vue

```diff
@@ -12,24 +12,41 @@
     </div>
 
     <div v-if="isUserLoggedIn == false" class="header__authsection">
-      <a aria-label="Login to account" class="header__authsection-login" @click="login"> Login </a>
-      <ButtonComponent label="Sign Up" ariaLabel="Sign up to account" name="signup_dataland_button" @click="register" />
+      <Button
+        label="LOGIN"
+        ariaLabel="Login to account"
+        name="login_dataland_button"
+        data-test="login-button"
+        @click="login"
+        icon="pi pi-user"
+        variant="text"
+        :pt="{
+          root: {
+            style: 'color: var(--p-orange-950);',
+          },
+          icon: {
+            style: 'color: var(--p-orange-950);',
+          },
+        }"
+      />
+      <Button label="SIGN UP" ariaLabel="Sign up to account" name="signup_dataland_button" @click="register" rounded />
     </div>
   </template>
   <template v-else>
     <div class="header__authsection">
-      <ButtonComponent
+      <Button
         label="Log in"
         buttonType="login-button"
         ariaLabel="Login to account"
```

Comment:
buttonType should do nothing here. Aria-label not needed.
```suggestion
label="Log in"
```